style.css

```diff
@@ -0,0 +1,1128 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: sans-serif;
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+.fa-chevron-right {
+  font-size: 1.25rem;
+  margin-left: 0.5rem;
+}
+
+h2 {
+  color: red;
+  font-size: 1.75rem;
+  font-weight: 900;
+  font-family: Avalon, sans-serif;
+  width: 100%;
+  margin-bottom: 0.5rem;
+}
+
+.lf-h h2 {
+  color: red;
+  font-size: 1.75rem;
+  font-weight: 900;
+  font-family: Avalon, sans-serif;
+  width: 100%;
+  margin-bottom: 0.5rem;
+}
+
+.lf-h-t a {
+  color: ghostwhite;
+  font-size: 1.25rem;
+  margin-left: 30%;
+}
+
+.flex {
+  display: flex;
+}
+
+#nav-bar-1 {
+  display: none;
+}
+
+#nav-bar-2 {
+  position: fixed;
+  background-color: white;
+  top: 0%;
+  width: 100%;
+  height: 100px;
+  z-index: 5;
+}
+
+#nav-bar-2-top {
+  justify-content: space-between;
+  width: 100%;
+  height: 3rem;
+}
+
+.emptyspace {
+  margin-right: 10px;
+  border-left: orange;
+  border-width: 2px;
+}
+
+.fa-search {
+  margin-top: 10px;
+  margin-right: 10px;
+  color: #555;
+}
+
+#nav-bar-2-bottom {
+  position: fixed;
+  width: 100%;
+  background-color: rgb(252, 224, 66);
+  height: 3.5rem;
+  font-size: 13px;
+  text-align: center;
+  padding-top: 1rem;
+}
+
+/* topic section */
+.todaytopic {
+  width: 100%;
+  margin-top: 6.5rem;
+  margin-left: 0.5rem;
+  margin-right: 8px;
+}
+
+.topic1 {
+  background-image: url(./pictures/label1.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 100%;
+  padding: 5px;
+  height: 110px;
+}
+
+.topic2,
+.topic3 {
+  margin-top: 10px;
+  background-image: url(./pictures/label2.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 100%;
+  padding: 5px;
+  height: 130px;
+}
+
+.topic3 {
+  background-image: url(./pictures/label3.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.topic1_d {
+  display: none;
+}
+
+.topic1_t {
+  font-size: 19px;
+  color: white;
+  font-weight: 900;
+  opacity: 1;
+  margin-top: 0.5rem;
+}
+
+.topic2_t {
+  font-size: 19px;
+  color: white;
+  font-weight: 900;
+  opacity: 1;
+  margin-top: 2.5rem;
+}
+
+.topic1_a {
+  font-size: 13px;
+  color: rgb(173, 173, 173);
+  margin-bottom: 1rem;
+}
+
+.grid {
+  display: grid;
+}
+
+/* latest news */
+#latestnews {
+  width: 90%;
+  margin: 10% auto 10% auto;
+}
+
+.lf-h-g img {
+  margin: 3rem 0 1rem 1rem;
+  height: 3.5rem;
+  width: 3.5rem;
+  background-color: #f33;
+}
+
+.lf-h-t img {
+  height: 3rem;
+  width: 3rem;
+  margin-top: 3rem;
+  margin-left: 0.5rem;
+}
+
+#ln-articles-container img {
+  max-width: 100%;
+  width: 5rem;
+  height: 5rem;
+}
+
+#ln-articles-container article {
+  display: flex;
+  width: 100%;
+  position: relative;
+  font-family: ARS Maquette, Helvetica Neue, Arial, sans-serif;
+  padding-bottom: 1rem;
+}
```

Comment:
- [ ] Please avoid using id selectors for styling, it's unconventional and is a [css bad practice](https://speckyboy.com/good-bad-css-practices/),. Kindly replace the id selectors in this milestone with class selectors.

Reply:
The ids used in css file were approved in previous milestone.